cypress/component/DarCollectionTable/researcher_actions.spec.js

```diff
@@ -64,53 +80,93 @@ describe('Researcher Actions - Container', () => {
-    const reviewButton = cy.get(`#researcher-review-${collectionId}`);
-    reviewButton.should('exist');
+    cy.get(`#revise-collection-${collectionId}`).should('exist');
+  });
+
+  it('does not render the revise button if there are elections present', () => {
+    propCopy.collection.dars = mixedCancelElectionDars;
+    mount(<ResearcherActions {...propCopy} />);
+    cy.get(`#revise-collection-${collectionId}`).should('not.exist');
   });
 });
 
-describe('Researcher Actions - Revise Button', () => {
-  it('renders the revise button if all of the DARs are cancelled', () => {
+describe('Researcher Actions - Review Button', () => {
+  it('renders the review button if the DAR has been submitted', () => {
+    propCopy.collection.dars = darsWithElections;
+    mount(<ResearcherActions {...propCopy} />);
+    cy.get(`#researcher-review-${collectionId}`).should('exist');
+  });
+
+  it('hides the review button if the collection is in draft status', () => {
+    propCopy.collection.isDraft = true;
+    mount(<ResearcherActions {...propCopy} />);
+    cy.get(`#researcher-review-${collectionId}`).should('not.exist');
+  });
+});
+
+describe('Researcher Actions - Resume Button', () => {
+  it('renders the resume button if the collection is in draft status', () => {
     propCopy.collection.dars = canceledDars;
+    propCopy.collection.isDraft = true;
     mount(<ResearcherActions {...propCopy} />);
-    const reviseButton = cy.get(`#revise-collection-${collectionId}`);
-    reviseButton.should('exist');
+    cy.get(`#researcher-resume-${collectionId}`).should('exist');
   });
 
-  it('does not render the revise button if there are open elections present', () => {
-    propCopy.collection.dars = darsWithElections;
+  it('hides the resume button if the collection is not in draft status', () => {
+    propCopy.collection.dars = {1: {data: {}}};
+    propCopy.collection.isDraft = false;
     mount(<ResearcherActions {...propCopy} />);
-    const reviseButton = cy.get(`#revise-collection-${collectionId}`);
-    reviseButton.should('not.exist');
+    cy.get(`#researcher-resume-${collectionId}`).should('not.exist');
+  });
+});
+
+describe('Researcher Actions - Cancel Button', () => {
+  it('renders the cancel button if the collection is not in draft status', () => {
+    propCopy.collection.dars = darsWithNoElections;
+    propCopy.collection.isDraft = false;
+    mount(<ResearcherActions {...propCopy} />);
+    cy.get(`#researcher-cancel-${collectionId}`).should('exist');
+  });
+
+  it('hides the cancel button if the collection is in draft status', () => {
+    propCopy.collection.dars = darsWithNoElections;
+    propCopy.collection.isDraft = true;
+    mount(<ResearcherActions {...propCopy} />);
+    cy.get(`#researcher-cancel-${collectionId}`).should('not.exist');
+  });
+
+  it('hides the cancel button if the collection already has elections created', () => {
+    it('hides the cancel button if the collection is in draft status', () => {
```

Comment:
nit: might be better to combine the nested `it` statements

Reply:
No that's just a bad copy-paste job. I have to remove the nested it block.